### ISSUE TITLE
Renaming minified and bundled files.

### ISSUE THREAD

**Issue by LarterStyles** [madskristensen/WebEssentials2013#1672]
If I rename a JavaScript file, the minified file isn't renamed accordingly. Similarly, if I rename a JavaScript file that's included as part of a bundle., the .bundle file isn't updated with the file's new name.

FYI: I'm using web forms project with Visual Studio 2013 Update 4 + the latest WE update.

Examples below:

![original](https://cloud.githubusercontent.com/assets/2833830/5160670/f1887f84-73cd-11e4-9586-85336da713e6.PNG)

![renamed](https://cloud.githubusercontent.com/assets/2833830/5160671/f5cd1cbc-73cd-11e4-9aca-0baf63323a7e.PNG)

**Comment by robvon** [comment]
I don't believe that the workaround above works in Vs2013. I suspect WebEssentials is monitoring the name of each '.bundle' file. If you change it - it fails to trigger the bundle process. Perhaps a file-watcher is being used utilised but not to watch renames...

Just reload the solution and WebEssentials spots the .bundle files correctly. Its not like you have to rename them often.
